### ISSUE TITLE
Icon cut off at bottom

### ISSUE THREAD

**Issue by kojow7** [lhz516/react-h5-audio-player#51]
For some reason my play button is getting cut off at the bottom. The relevant portion of my code:

`                <div style={{backgroundColor: "red", height: "200px"}}>
                                          <AudioPlayer
                                            autoPlay
                                            src="http://example.com/file.mp3"
                                            onPlay={e => console.log("onPlay")}
                                            // other props here
                                          />
                </div>
`
Any ideas why this might be? At first I thought it was my parent container. However, I have now enlarged that as shown in the attached picture but still have the same problem.

![audio_h5](https://user-images.githubusercontent.com/13071962/83696288-4e55cc80-a5b9-11ea-9fe2-5c9562535851.png)

**Comment by lhz516** [comment]
Hi! Thanks for reporting. Please provide more info:

Package version:
React version:
Browser and its version:
OS and its version:

Does the icon in storybook example also get cut? https://static.hanzluo.com/react-h5-audio-player-storybook/index.html?path=/docs/layouts--default-story

Also, did you check if there are any global styles that overwriting the player's?

**Comment by kojow7** [comment]
I am using the following:
react-h5-audio-player: ^3.1.0
react: ^16.4.0
Chrome version:  83.0.4103.97
OS version: Windows 10

It also has this problem on MIcrosoft Edge version: 44.18362.449.0

However, on a Mac OS, there are no problems on either Safari or Chrome 81.0.4044.138

In looking at the "inspect" tool, I have discovered that the SVG element is being supplied some values by Bootstrap.

```
svg {
    overflow: hidden;
    vertical-align: middle;
}
```
When I change "middle" to another value such as "unset" or "initial" then it is no longer cut off.

I was using a local Boostrap 4.1.3 min file and found this issue for Boostrap 4.1.2: https://github.com/twbs/bootstrap/issues/26878 (though I do not think it applies to 4.1.3).

I have now removed my own Bootstrap version and am direct linking to Boostrap 4.5. However, even though I updated, the issue still exists in both Chrome and Edge for Windows, but not on Safari or Chrome for MacOS.

**Comment by lhz516** [comment]
I see. This project uses flexbox to control vertical alignment which is sort of incompatible with `vertical-align`. In next version, I'll add `vertical-align: initial` to the player's `svg`s.

For now, you can use the hack below:
```css
.rhap_container svg {
  vertical-align: initial;
}
```

**Comment by kojow7** [comment]
Yes, the hack works well. Thank you. Any idea why the Windows browsers (Chrome / Edge) had the issue, but the MacOS browsers (Chrome / Safari) didn't?

**Comment by lhz516** [comment]
@kojow7 I think it's a bug of Windows browsers. Even if it's vertically middle aligned, there shouldn't be any issues visually. Anyways, the overwrite will fix it.

**Comment by lhz516** [comment]
Published 3.2.0

Closing
